easybuild/easyconfigs/m/makedepend/makedepend-1.0.5-foss-2016a.eb

```diff
@@ -10,7 +10,11 @@ toolchain = {'name': 'foss', 'version': '2016a'}
 toolchainopts = {'optarch': True}
 
 sources = [SOURCE_TAR_GZ]
-source_urls = [XORG_UTIL_SOURCE]
+source_urls = ['http://xorg.freedesktop.org/releases/individual/util']
```

Comment:
I just started by using one of the other `makedepend` eb config files as a template and this is what was contained in there...